pkgs/applications/audio/dragonfly-reverb/default.nix

```
@@ -6,8 +6,8 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "michaelwillis";
     repo = "dragonfly-reverb";
-    rev = "1.0.0";
-    sha256 = "05m4hd8lg0a7iiia6cbiw5qmc4p8vbkxp2qh7ywaabawiwa9r24x";
+    rev = "1.1.4";
```

You can now use `pname = "dragonfly-reverb"` and `version = "1.1.4"`. Nix will automatically concatenate them.
Then you can do:
```suggestion
    rev = version;
```

and also `repo = pname`